time_weighted_average_price/attest_fn_call/Makefile

```diff
@@ -7,7 +7,10 @@ check:
 	@command -v bky-as >/dev/null 2>&1 || { echo >&2 "bky-as not found in PATH. Aborting."; exit 1; }
 	@command -v jq >/dev/null 2>&1 || { echo >&2 "jq not found in PATH. Aborting."; exit 1; }
 
-tmp/x.wasm: main.go $(wildcard price/*.go) | tmp check
+SRCS := $(shell find . -name '*.go')
```

Comment:
I'm using `find` here to grab the go files from this directory and any subdirectories, like `price`. I'm using the same pattern in all other makefiles for consistency